src/GeositeFramework/css/main.css

```diff
@@ -1553,6 +1553,7 @@ header .nav-main {
 .control-container {
     position: absolute;
     width: 100%;
+    z-index: 102;
```

Comment:
For reference it seems like this is why we're using 102 here:

https://github.com/CoastalResilienceNetwork/GeositeFramework/blob/ea036a0d3c9051bdd2876fcf48187b45db9aa10e/src/GeositeFramework/css/main.css#L1581